README.md

```diff
@@ -3026,7 +3026,6 @@ Other Style Guides
     // bad
     import CheckBox from './check_box'; // PascalCase import/export, snake_case filename
     import forty_two from './forty_two'; // snake_case import/filename, camelCase export
-    import inside_directory from './inside_directory'; // snake_case import, camelCase export
```

Comment:
this line deals with a snake_case import.

Reply:
I'm sorry, I did not make it clear.
I think this lint:
```javascript
  import forty_two from './forty_two';  // snake_case import/filename, camelCase export
```
and this line:
```javascript
  import inside_directory from './inside_directory'; // snake_case import, camelCase export
```

Actually, they both snake_case import/filename, camelCase export. 🙃

Reply:
Both demonstrate different things; one is a directory import, the other is a non-directory import.

Reply:
Wow, wow, It's my fault, I'm so careless! Thank you.

Reply:
No problem at all; thanks for contributing!